### ISSUE TITLE
Wrong location when using UIFindInteraction

### ISSUE THREAD

**Issue by Blankwonder** [simonbs/Runestone#215]
When using the iOS 16 UIFindInteraction, the text view always scrolls to an incorrect offset if the target text range is not visible.

Here is a screen recording. (Tested on iOS 16.1 beta 4)

https://www.dropbox.com/s/35hlkfpqsmp0muo/RPReplay_Final1665378953.MP4?dl=0

**Comment by simonbs** [comment]
@Blankwonder Once again thanks for opening the issue. I have had a look at this issue and while I was unable to reproduce the exact issue you describe, I did find an issue where the search results may not be correctly highlighted. This should be fixed by the changes in #216.

If the search results aren't properly highlighted, it may be difficult to determine if Runestone is scrolling to the correct position, in particular when dealing with long texts.

Another ting to keep in mind is that Runestone (and UITextView) will only scroll to the correct position when an on-screen keyboard is shown if the content insets of the text view has been updated to account for the keyboard. If that is not the case, then the search result will appear below the keyboard.

Can I get you to verify that you are accounting for the on-screen keyboard and test again with the main branch where #216 is been merged in?
